spec/fixtures/snapshots/csv_imports/coin-metro-transactions.csv.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "Test Wallet",
+  "txn_count": 96,
+  "entry_count": 197,
+  "balances": {
+    "EUR": "-14997.02120375",
+    "XCM": "-216306.9496725"
```

The balances are incorrect, they should be:
```suggestion
    "EUR": "-22.62879625",
    "XCM": "478415.9216725"
```

---

Recalculated